### PR TITLE
feat(style/skate): water and green space color changes

### PR DESCRIPTION
**Pull request by firestack**
Asana Ticket: https://app.asana.com/0/1203014709808707/1203180633835790/f

# Summary
- greenspace color #b1deaf
- greenspace use white halo
- water color #b6bee7
- water labels use #2c45c6

Make style more consistent and clean up trailing whitespace.

## Links
The first three commits are whitespace and code style fixes, browsing the last 4 commits will narrow down the scope of what actually changed.
https://github.com/mbta/tile-server/pull/25/files/16661ad191f3e4c2c5eb39d484ed417c0a98c70b..0cc592a1a5da6290090ba9c5de2a99f25af55578

### CODE REVIEW

File: etc/skate-style/palette.mss [hunk 1]
```diff
@@ -39,22 +39,25 @@ Map { font-directory: url(./fonts); }
...
+@wooded:            @greenspace;
+@agriculture:       #f2e8b6;
 
 @building:          @background_grey;
 @building_line:     #dbdde1;
 @building_case:     #d3d1c8;
 @hospital:          #ebe3da;
 @school:            @background_grey;
-@sports:            #c9e2c1;
+@sports:            @greenspace;
```
Comment (firestack) [inline etc/skate-style/palette.mss:60]: I wish to move the "skate color groups" to their own file or area, but for roughing it out I just added it inline, I'd also group the "greenspace" variables for clarity, I'd appreciate comments on that if anyone has strong opinons.

File: etc/skate-style/palette.mss [hunk 2]
```diff
@@ -129,6 +132,9 @@ Map { font-directory: url(./fonts); }
    at once or override each individually. */
 @place_halo:        fadeout(#fff,34%);
 
+@halo_white:        @place_halo;
+@greenspace_halo:   @halo_white;
+
```
Comment (firestack) [inline etc/skate-style/palette.mss:137]: This also doesn't feel like the final resting place of these variables, so let me know if you have ideas of where it should be organized into.

**Comment by lemald** [comment]
**praise:** thank you for cleaning up a bunch of the weird whitespace in these style files. Or maybe your editor did it automatically, but in any event I like it.

**Review by lemald — APPROVED** [review]
👍

**Pull request merged**